rskj-core/src/test/java/co/rsk/peg/federation/constants/FederationConstantsTest.java

```diff
@@ -98,10 +98,9 @@ void test_getFederationActivationAge(FederationConstants federationConstants, bo
 
     private static Stream<Arguments> getGenesisFederationCreationTimeTestProvider() {
         return Stream.of(
-            Arguments.of(BridgeMainNetConstants.getInstance(), Instant.ofEpochMilli(1514948400L)),
-            Arguments.of(BridgeTestNetConstants.getInstance(), Instant.ofEpochMilli(1538967600L)),
-            Arguments.of(new BridgeRegTestConstants(), Instant.ofEpochSecond(1451606400L)),
-            Arguments.of(BridgeDevNetConstants.getInstance(), Instant.ofEpochMilli(1510617600L))
```

Comment:
Why `BridgeRegTestConstants` is not being mocked anymore?

Reply:
It wan't mocked, it was obtained via `getInstance` method and now it's uilt via its constructor